core/src/main/scala/org/apache/spark/executor/ExecutorSource.scala

```diff
@@ -75,6 +80,7 @@ class ExecutorSource(threadPool: ThreadPoolExecutor, executorId: String) extends
 
   // Expose executor task metrics using the Dropwizard metrics system.
   // The list is taken from TaskMetrics.scala
+  val SUCCESSFUL_TASKS = metricRegistry.counter(MetricRegistry.name("successfulTasks"))
```

Comment:
This is successfulTasks, and on Line57 is startedTasks, are they the same?

Reply:
The two metrics come from different sources, one from the Java thread pool and the other from the executor. When all runs soomthly the values of the two metrics should be very close/the same. Large differences between those 2 values can help spotting issues.